code/integration-tests/runtime-tests/tests/utils/ibcUtils.ts

```diff
@@ -0,0 +1,306 @@
+    preBalanceOnPicasso,
+    feeAccountPreBalance,
+    preBalancesOnOsmosis
+  };
+}
+
+export async function getStateOfFundsForPicassoToComposable(
+  composableApi: ApiPromise,
+  picassoApi: ApiPromise,
+  wallet: KeyringPair
+) {
+  const preTotalIssuanceOnComposable = await queryTotalIssuanceOncomposable(composableApi);
+  const preTotalIssuanceOnPicasso = await queryTotalIssuanceOfTokensOnPicasso(picassoApi);
+  const preBalancesOnComposable = await queryAllBalanceOnComposable(
+    composableApi,
+    [picaIdOnComposable, ksmOnComposable, dotOncomposable, usdtOnComposable],
+    wallet.address);
+  const preBalanceOnPicasso = await queryAllBalanceOnPicasso(picassoApi, [1, 4, 6, 130], wallet.address);
+  const feeAccountPreBalance = await queryAllBalanceOnPicasso(picassoApi, [1, 4, 6, 130], picassoFeeAddress);
+  return {
+    preTotalIssuanceOnComposable,
+    preTotalIssuanceOnPicasso,
+    preBalancesOnComposable,
+    preBalanceOnPicasso,
+    feeAccountPreBalance
+  };
+}
+
+export async function queryAllBalanceOnComposable(api: ApiPromise, assets: string[], walletAddress: string) {
+  const retObject = {'1': '', '4': '0', '6': '0', '130': '0'};
+  const objKeys = Object.keys(retObject);
+  await Promise.all(assets.map(async (assetId, index) => {
+    const balance = await api.query.tokens.accounts(walletAddress, assetId);
+    retObject[objKeys[index] as keyof IPreBalancesOnPicasso] = balance.free.toString();
+  }));
+  return retObject;
+}
+
+export async function queryTokenBalance(api: ApiPromise, walletAdress: string, assetId: string) {
+  const {free} = await api.query.tokens.accounts(walletAdress, assetId);
+  return free.toString();
+}
+
+export function createIbcTxsForBatch(
+  api: ApiPromise, amounts: string[], assets: number[] | string [], receiver: string, toChain: string): SubmittableExtrinsic<"promise">[] {
+  let channelId: number;
+  let memo = '';
+  if (toChain === 'centauri') {
+    channelId = 0;
+  } else if (toChain === 'picasso') {
+    channelId = 0;
+  } else if (toChain == 'osmosis') {
+    channelId = 0;
+    memo = `{"forward":{"receiver":"${osmosisAddress}","port":"transfer","channel":"channel-3","timeout":600000000000,"retries":0}}`;
+  } else {
+    channelId = 1;
+  }
+  const txsArr = assets.map((asset, index) => {
+    return createIbcTransferExtrinsic(api, receiver, asset, amounts[index], channelId, toChain, memo);
+  })
+  return txsArr;
+}
+
+export async function waitForSeconds(seconds: number) {
+  return new Promise(resolve => {
+    setTimeout(resolve, seconds * 1000)
+  })
+}
+
+export async function queryBalancesOnOsmosis(address: string){
+  const {stdout} = await exec(`~/go/bin/osmosisd query bank balances ${address} --output json`);
+  const formattedBalance = JSON.parse(stdout);
+  const balances = parseBalancesFromOsmosis(formattedBalance.balances);
+  return balances;
+}
+
+export async function parseBalancesFromOsmosis(balances: []){
+  const retBalances = {
+    "dot": '0',
```

Comment:
remove DOT USDT, etc. do not use symbols. at least do not use Symbols for main most stable impotatn test.

Reply:
symbols are offchain to be UI tested.

Reply:
I have modified the way that I retrieve balances and these methods were actually obsolete but accidentally committed the old version. So just removed all obsolete methods that uses these.